spec/serializers/upload_serializer_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe UploadSerializer do
+  let(:upload) { Fabricate(:upload) }
+
+  it 'should render without errors' do
+    expect { UploadSerializer.new(upload).to_json }.not_to raise_error
```

Comment:
`not_to raise_error` is an anti-pattern here because the test will just fail with the error if an error was raised. I will just test for some simple fields returned by the serializer here.